docs/guides/qiskit-addons-sqd.mdx

```diff
@@ -0,0 +1,60 @@
+description: Overview of the Sample-based quantum diagonalization (SQD) workflow
+---
+
+# Sample-based quantum diagonalization (SQD)
+
+Sample-based quantum diagonalization (SQD) is a technique for finding eigenvalues and eigenvectors of quantum operators, such as the Hamiltonian of a quantum system, using quantum and distributed classical computing together.
+
+Classical computing is used to process samples obtained from a quantum processor, and to project and diagonalize a target Hamiltonian in a subspace spanned by them. This allows SQD to be robust to samples corrupted by quantum noise and manage large Hamiltonians, such as chemical systems with millions of interacting terms, beyond the reach of exact diagonalization methods.
+
+The SQD tool can target Hamiltonians expressed as linear combinations of Pauli operators, or second-quantized fermionic operators. The input samples are obtained by quantum circuits defined by the user, which are believed to be good representations of eigenstates (for example the ground state) of a target operator. The convergence rate of SQD as a function of the number of samples improves with the sparseness of the target eigenstate.
+
+## Install SQD package
+
+There are two ways to install the SQD package. Via PyPI and building from source. We also recommend installing these packages in a [virtual environment](https://docs.python.org/3.10/tutorial/venv.html) to ensure separation between package dependencies.
+
+### Install from PyPI
+
+The most straightforward way to install the `qiskit-addon-sqd` package is via PyPI.
+
+```bash
+pip install qiskit-addon-sqd
+```
+
+### Build from source
+
+Users who wish to contribute to this package or who want to install it manually may do so by first cloning the repository
+
+```bash
+git clone git@github.com:Qiskit/qiskit-addon-sqd.git
+```
+
+and install the package via `pip`. The repository also contains example notebooks which can be run and if you plan on developing in the repository, you may want to also install the `dev` dependencies.
+
+Adjust the options to suit your needs.
+
+```bash
+pip install tox notebook -e '.[notebook-dependencies, dev]'
+```
+
+## Theoretical background
+
+The SQD workflow using self-consistent configuration recovery is explained in detail in [[1]](#references). This page will briefly overview the technique depicted in the following diagram.
+
+![SQD diagram depicting configuration recovery, collecting subsamples, and obtaining eigenstates from those subsamples](/images/guides/qiskit-addons/sqd_diagram.png)
+
+Here $\bar{\mathcal{X}}$ is a set of noisy samples containing, in the context of the Hamiltonian being simulated, physical and non-physical configurations (represented as bitstrings) obtained from execution on a QPU. The non-physical configurations are due to noise and can be processed by the `sqd.configuration_recovery.recover_configurations()` method to refine the samples into a new set $\mathcal{X}_R$. 
```

Comment:
I think it would be good to emphasise somewhere in this section that the noisy samples come from a hardware run, and every step in the diagram after that is purely classical

Reply:
> also i know the api ref isn't ready yet but if we're going to mention the `recover_configurations()` function by name we should probably link to some more info about how to use the code

Yeah I was planning on adding a link in once the API docs have been migrated.



> I think it would be good to emphasize somewhere in this section that the noisy samples come from a hardware run, and every step in the diagram after that is purely classical

I'll add a point in here about these samples come from a QPU.